metadata.json

```diff
@@ -39,6 +39,13 @@
         "7",
         "8"
       ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "18.04",
```

Comment:
Maybe you'll want to add 16.04 here to match syslog-ng or drop this EOL release in the other module?